settings.gradle

```diff
@@ -225,8 +225,9 @@ include ":benchmarks",
         ":examples:kotlin-jsr223-local-example",
         ":examples:kotlin-jsr223-daemon-local-eval-example",
         ":kotlin-annotations-jvm",
-        ":kotlin-scripting-common",
-        ':kotlin-scripting-js',
+        ":kotlin-scripting-common"
```

Comment:
I think this is an irrelevant change.
If you need to split includes for some reasons, please do it in a separate commit with an explanation.

Reply:
It was done because groovy supports only a limited number of arguments for `include` method, and adding +1 exceeded this number. Somebody splitted it for me, so I can revert it back